### PR TITLE
fix(ios): correctly decode device token for ios 13 compatibility

### PR DESCRIPTION
**Pull request by ewanharris**
Fixes TIMOB-27386

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27386

### CODE REVIEW

File: iphone/Classes/NetworkModule.m [hunk 1]
```diff
@@ -286,11 +286,8 @@ - (void)application:(UIApplication *)application didRegisterForRemoteNotificatio
 {
   // called by TiApp
   if (pushNotificationSuccess != nil) {
-    NSString *token = [[[[deviceToken description] stringByReplacingOccurrencesOfString:@"<" withString:@""]
-        stringByReplacingOccurrencesOfString:@">"
-                                  withString:@""]
-        stringByReplacingOccurrencesOfString:@" "
-                                  withString:@""];
+    NSString *token = [NSString hexStringFromData:deviceToken];
```
Comment (vijaysingh-axway) [inline iphone/Classes/NetworkModule.m]: There is no such static method on NSString "hexStringFromData:" .

File: iphone/Classes/TiApp+Addons.m [hunk 2]
```diff
@@ -154,11 +154,8 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
 
 - (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken
 {
-  NSString *token = [[[[deviceToken description] stringByReplacingOccurrencesOfString:@"<" withString:@""]
-      stringByReplacingOccurrencesOfString:@">"
-                                withString:@""]
-      stringByReplacingOccurrencesOfString:@" "
-                                withString:@""];
+
+  NSString *token = [NSString hexStringFromData:deviceToken];
```
Comment (vijaysingh-axway) [inline iphone/Classes/TiApp+Addons.m]: Same as above

**Review by vijaysingh-axway — CHANGES_REQUESTED** [review]

File: iphone/Classes/NetworkModule.m (hunk 1 above)
Comment (hansemannn) [reply iphone/Classes/NetworkModule.m]: Please use the following:
```objc
const unsigned *tokenBytes = [deviceToken bytes];
NSString *stringToken = [NSString stringWithFormat:@"%08x%08x%08x%08x%08x%08x%08x%08x",
                         ntohl(tokenBytes[0]), ntohl(tokenBytes[1]), ntohl(tokenBytes[2]),
                         ntohl(tokenBytes[3]), ntohl(tokenBytes[4]), ntohl(tokenBytes[5]),
                         ntohl(tokenBytes[6]), ntohl(tokenBytes[7])];
 ```

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by ewanharris** [comment]
Closing as I borked this, will cherry pick and open a new PR

**Pull request closed without merging**